### DISCUSSION TITLE
Very low volume from NoiseTorch microphone

### DISCUSSION THREAD

**Discussion by Doug140** [noisetorch/NoiseTorch#381] [Q&A (How can I ...?)] [answered]
About a week ago I started to get very low volume when recording with the NoiseTorch microphone selected. Before that, the volume levels were about the same from the normal microphone and the virtual NoiseTorch mic. I don't know what might have changed. Does anyone have any ideas about why this is happening?

The attached zip file contains a short recording where you can hear the difference.
(https://github.com/noisetorch/NoiseTorch/files/10361516/regular.vs.noisetorch.mic.compariason.zip)

 Here's my pulseAudio setup:
(https://github.com/noisetorch/NoiseTorch/files/10361493/regular.vs.noisetorch.mic.compariason.zip)
![Screenshot_20230106_101451](https://user-images.githubusercontent.com/63794602/211053351-81e342f0-984f-48cf-90ff-e12340522db7.png)

**Comment by Doug140** [comment, marked as answer]
I think I got my problem solved. I looked at the troubleshooting section here:
https://github.com/noisetorch/NoiseTorch/wiki/Troubleshooting

While my noisetorch mic isn't completely silent, the troubleshooting article gave me the idea to look at the settings for "Null Output" and "LADSPA Plugin Noise Suppressor for Voice (Mono) on Null Output". In my case, both of these were at about 40%. I moved them to 100%, and now I get normal output from the noisetorch mic.

I don't have enough technical knowledge to know WHY that worked. Can someone explain it to me?

**Reply by ZyanKLee** [reply]
noisetorch re-routes the audio signals from your microphone through those two devices to inject the rnnoise filter. When those devices are not at 100%, each reduces the overall volume by the amount they are reduced individually. So in your case this was effectively 40% of 40% of the original volume which is set to 53% for the webcam in your screenshot - resulting in ~8.5%. (math: 0.53x0.4x0.4=0.0848)

**Reply by Doug140** [reply]
Thank you for the explanation. I don't know how those levels got changed, but if I have a similar problem in the future I'll know where to look.
